src/AdvApi32.Desktop/AdvApi32.cs

```diff
@@ -19,14 +19,169 @@ public static partial class AdvApi32
         /// because services and service groups share the same name space.
         /// </summary>
         public const string SC_GROUP_IDENTIFIER = "+";
-        public const uint INFINITE = 0xFFFFFFFF;
+        public const uint SERVICE_NO_CHANGE = 0xFFFFFFFF;
```

Comment:
I wouldn't change the enum base types, no. They don't tend to get in the way of users either way, and leaving their base types as `uint` means you don't have to make unchecked casts for every single enum value definition.